include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -816,11 +837,22 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", []> {
         for(int k = 0; k < 8 /*size_0*/; k++)
           // access/store element at/to index (i * 16 /*stride_2*/ + j * 1 /*stride_1*/ + k * 2 /*stride_0*/)
     ```
+
+    ## Important gotcha regarding strides
+
+    All strides are expressed in multiples of the element width (just like `len` and `offset`)
+    **with the caveat that the inner-most dimension's stride must be a multiple of 32b**.
+    I.e., `(stride_0 * element_width) % 32 == 0`. For example, if you are striding a `memref<512xi16>`,
+    `stride_0` cannot be 1 (or 3, or 5, ...). The reason for this requirement is stream access to DMA memory is 32b
```

Comment:
rather than saying "cannot", I would say "stride_0" must be a multiple of 2.  if the element type is i8, then similarly the stride parameter must be a multiple of 4.

Reply:
this is wrong now.